### PR TITLE
Add jest extender to the image and build commands

### PR DESCRIPTION
**Pull request by sneridagh**

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -59,14 +59,17 @@ if [ ! -d "/opt/frontend/my-volto-project/src/addons/$GIT_NAME" ]; then
   cd /opt/frontend/my-volto-project/
 fi
 
-sed -i "s#\$GIT_NAME#$GIT_NAME#g" jest.config.js
 node /jsconfig $PACKAGE addons/$GIT_NAME/src
 
+if [ ! -d "/opt/frontend/my-volto-project/src/addons/$GIT_NAME/jest-addon.config.js" ]; then
```
Comment (avoinea) [inline docker-entrypoint.sh:64]: I don't this condition is correct, right now it tests if the directory `..jest-addon.config.js` doesn't exist.

File: docker-entrypoint.sh [hunk 2]
```diff
@@ -59,14 +59,17 @@ if [ ! -d "/opt/frontend/my-volto-project/src/addons/$GIT_NAME" ]; then
...
+  cp /opt/frontend/my-volto-project/src/addons/$GIT_NAME/jest-addon.config.js /opt/frontend/my-volto-project/.
+fi
+
 yarn
 
 if [[ "$1" == "test"* ]]; then
   yarn add -W --dev jest-junit jest-transform-stub
-  exec bash -c "set -o pipefail; ./node_modules/jest/bin/jest.js --env=jsdom --passWithNoTests src/addons/$GIT_NAME --watchAll=false --reporters=default --reporters=jest-junit --collectCoverage --coverageReporters lcov cobertura text 2>&1 | tee -a unit_tests_log.txt"
+  exec bash -c "set -o pipefail; RAZZLE_JEST_CONFIG=jest-addon.config.js yarn test src/addons/$GIT_NAME --watchAll=false --reporters=default --reporters=jest-junit --collectCoverage --coverageReporters lcov cobertura text 2>&1 | tee -a unit_tests_log.txt"
```
Comment (avoinea) [inline docker-entrypoint.sh:72]: For consistency and to be able to override this env should be handled before: 

```bash
if [ -z "$RAZZLE_JEST_CONFIG" ]; then
  RAZZLE_JEST_CONFIG="jest-addon.config.js"
fi
```

File: Dockerfile [hunk 3]
```diff
@@ -16,7 +16,6 @@ USER node
 RUN yo --no-insight @plone/volto my-volto-project --no-interactive
 
 COPY jsconfig docker-entrypoint.sh /
-COPY jest.config.js /opt/frontend/my-volto-project/
```
Comment (avoinea) [inline Dockerfile:19]: We can have an empty default `jest-addon.config.js` here in case the add-on doesn't provide it.

**Review by avoinea — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 3 above)
Comment (sneridagh) [reply Dockerfile:19]: Done

File: docker-entrypoint.sh (hunk 2 above)
Comment (sneridagh) [reply docker-entrypoint.sh]: Done

File: Dockerfile (hunk 3 above)
Comment (sneridagh) [reply Dockerfile:19]: Done

**Review by avoinea — APPROVED** [review]

**Pull request merged**